utils/build_and_install_openmpi.sh

```diff
@@ -5,5 +5,17 @@
 set -xe
 
 RECIPES_DIR=$(cd $(dirname $0)/../recipes && pwd -P)
-conda build ${RECIPES_DIR}/openmpi -c defaults -c conda-forge
-conda install openmpi --use-local --yes
+conda build ${RECIPES_DIR}/openmpi -c defaults -c conda-forge -c conda-forge/label/cf201901
```

Comment:
Are there any error messages that motivated adding ` -c conda-forge/label/cf201901`?

Reply:
Yes, the openmpi installation tries to install toolchain, which is not available in the other channels.